WORKFLOW.md

```diff
@@ -0,0 +1,100 @@
+# Development Workflow
+
+## Overview ✌🏻
+
+This guide provides information about:
+
+* Cleaning and setup
+* New sprints, iterations or particular features
+* Prepare code base releases
+* Update working branches after release
+* Add fixes or hot-fixes on release and master branches
+
+## Workspace 🧽
+
+Before start to work on an issue as a developer you need to ensure that your local repository is up to date.
+
+First, is necessary to update your *develop* branch to match remote changes.
+
+```shell
+git checkout develop
+```
+
+```shell
+git fetch origin
+```
+
+```shell
+git pull origin develop
+```
+
+Finally, you must clean remote references to branches:
```

Comment:
Why is it relevant for us, they to do this?

Reply:
Just a best practice, but we are not closed to other approaches 😬